larray/inout/pandas.py

```diff
@@ -291,9 +293,27 @@ def _series_to_group(series, axis):
     return LGroup(key=series.values, name=name, axis=axis)
 
 
-# ################################## #
-#    DATAFRAME <--> AXES, GROUPS     #
-# ################################## #
+def _metadata_to_series(metadata):
+    s = pd.Series(data=list(metadata.values()), index=metadata.keys(), name='')
+    s.index.name = 'METADATA'
+    return s
+
+
+def _series_to_metadata(series):
+
+    def _convert_value(key, value):
+        value = str(value)
+        value = pd.to_numeric([value], errors='ignore')[0]
```

Comment:
why do you use [value] instead of just value? pd.to_numeric seems to handle scalars just fine (even if that is undocumented)

Reply:
What version of pandas do you have? If I try to pass value instead of [value], I get the following error message: TypeError: 'int' object is not subscriptable

Reply:
```python
>>> pd.to_numeric('10', errors='ignore')
10
>>> pd.to_numeric(10, errors='ignore')
10
>>> pd.__version__
'0.20.3'
```